### PR TITLE
add code style check scripts to travis-ci

### PR DESCRIPTION
**Pull request by cyjseagull**
add code style check scripts to travis-ci
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/wheatli%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/23206367%3Fv%3D4%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/FISCO-BCOS/lab-bcos/pull/7%23issuecomment-415362809%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222018-08-23T10%3A04%3A53Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/23206367%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/wheatli%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%205160d74c14b2a1cb1e81fdc04db87fa3dc7bfb68%20tools/ci/check-commit.sh%20121%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/FISCO-BCOS/lab-bcos/pull/7%23discussion_r212208362%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22One%20empty%20line%20in%20the%20end%20is%20enough%22%2C%20%22created_at%22%3A%20%222018-08-23T07%3A33%3A39Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/23206367%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/wheatli%22%7D%7D%2C%20%7B%22body%22%3A%20%22done%22%2C%20%22created_at%22%3A%20%222018-08-23T10%3A45%3A34Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/4343461%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cyjseagull%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222018-08-23T10%3A46%3A43Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/4343461%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cyjseagull%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20tools/ci/check-commit.sh%3AL1-123%22%7D%2C%20%22Pull%205160d74c14b2a1cb1e81fdc04db87fa3dc7bfb68%20tools/ci/run-clang-format.py%208%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/FISCO-BCOS/lab-bcos/pull/7%23discussion_r212208042%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22pls%20add%20copyright%20of%20this%20file%2C%20like%20this%20%5C%22%20This%20file%20is%20download%20from%20https%3A//github.com/Sarcasm/run-clang-format%5C%22%22%2C%20%22created_at%22%3A%20%222018-08-23T07%3A32%3A17Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/23206367%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/wheatli%22%7D%7D%2C%20%7B%22body%22%3A%20%22the%20copyright%20description%20has%20been%20added%22%2C%20%22created_at%22%3A%20%222018-08-23T09%3A28%3A04Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/4343461%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cyjseagull%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222018-08-23T10%3A04%3A22Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/23206367%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/wheatli%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20tools/ci/run-clang-format.py%3AL1-326%22%7D%2C%20%22Pull%205160d74c14b2a1cb1e81fdc04db87fa3dc7bfb68%20tools/ci/check-commit.sh%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/FISCO-BCOS/lab-bcos/pull/7%23discussion_r212208689%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Pls%20check%20these%20comments%2C%20is%20it%20right%3F%3F%22%2C%20%22created_at%22%3A%20%222018-08-23T07%3A34%3A48Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/23206367%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/wheatli%22%7D%7D%2C%20%7B%22body%22%3A%20%22already%20modify%20this%20description%20in%20https%3A//github.com/FISCO-BCOS/lab-bcos/pull/7/commits/edd15e557faf05912e0deb0a08b1ab81b8a6f448%22%2C%20%22created_at%22%3A%20%222018-08-23T10%3A46%3A26Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/4343461%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cyjseagull%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-complete%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222018-08-23T10%3A46%3A43Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/4343461%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cyjseagull%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20tools/ci/check-commit.sh%3AL1-123%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/FISCO-BCOS/lab-bcos/pull/7%23issuecomment-415362809%22%2C%20%22https%3A//github.com/FISCO-BCOS/lab-bcos/pull/7%23discussion_r212208042%22%2C%20%22https%3A//github.com/FISCO-BCOS/lab-bcos/pull/7%23discussion_r212208362%22%2C%20%22https%3A//github.com/FISCO-BCOS/lab-bcos/pull/7%23discussion_r212208689%22%2C%20%22https%3A//github.com/FISCO-BCOS/lab-bcos/pull/7%23discussion_r212241123%22%2C%20%22https%3A//github.com/FISCO-BCOS/lab-bcos/pull/7%23discussion_r212252361%22%2C%20%22https%3A//github.com/FISCO-BCOS/lab-bcos/pull/7%23discussion_r212263162%22%2C%20%22https%3A//github.com/FISCO-BCOS/lab-bcos/pull/7%23discussion_r212263353%22%2C%20%22https%3A//github.com/FISCO-BCOS/lab-bcos/pull/7%23discussion_r212263420%22%2C%20%22https%3A//github.com/FISCO-BCOS/lab-bcos/pull/7%23discussion_r212263424%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/wheatli'></a>

- [x] <a href='#crh-comment-Pull 5160d74c14b2a1cb1e81fdc04db87fa3dc7bfb68 tools/ci/run-clang-format.py 8'></a> &nbsp;<b><a href='https://github.com/FISCO-BCOS/lab-bcos/pull/7#discussion_r212208042'>File: tools/ci/run-clang-format.py:L1-326</a></b>
- <a href='https://github.com/wheatli'></a> pls add copyright of this file, like this " This file is download from https://github.com/Sarcasm/run-clang-format"
- <a href='https://github.com/cyjseagull'></a> the copyright description has been added
- [x] <a href='#crh-comment-Pull 5160d74c14b2a1cb1e81fdc04db87fa3dc7bfb68 tools/ci/check-commit.sh 121'></a> &nbsp;<b><a href='https://github.com/FISCO-BCOS/lab-bcos/pull/7#discussion_r212208362'>File: tools/ci/check-commit.sh:L1-123</a></b>
- <a href='https://github.com/wheatli'></a> One empty line in the end is enough
- <a href='https://github.com/cyjseagull'></a> done
- [x] <a href='#crh-comment-Pull 5160d74c14b2a1cb1e81fdc04db87fa3dc7bfb68 tools/ci/check-commit.sh 5'></a> &nbsp;<b><a href='https://github.com/FISCO-BCOS/lab-bcos/pull/7#discussion_r212208689'>File: tools/ci/check-commit.sh:L1-123</a></b>
- <a href='https://github.com/wheatli'></a> Pls check these comments, is it right??
- <a href='https://github.com/cyjseagull'></a> already modify this description in https://github.com/FISCO-BCOS/lab-bcos/pull/7/commits/edd15e557faf05912e0deb0a08b1ab81b8a6f448

<a href='https://www.codereviewhub.com/FISCO-BCOS/lab-bcos/pull/7?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/FISCO-BCOS/lab-bcos/pull/7?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/FISCO-BCOS/lab-bcos/pull/7?approve=0'></a>&nbsp;<a href='https://github.com/FISCO-BCOS/lab-bcos/pull/7'></a>
<a href='#crh-end'></a>

### CODE REVIEW

File: tools/ci/run-clang-format.py [hunk 1]
```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/env python
+"""A wrapper script around clang-format, suitable for linting multiple files
+and to use for continuous integration.
+
+This is an alternative API for the clang-format command line.
+It runs over multiple files and directories in parallel.
+A diff output is produced and a sensible exit code is returned.
+
```
Comment (wheatli) [inline tools/ci/run-clang-format.py]: pls add copyright of this file, like this " This file is download from https://github.com/Sarcasm/run-clang-format"

File: tools/ci/check-commit.sh [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        exit 0
+    else
+        echo "######### ERROR: Format check failed, please adjust them before commit"
+        exit 1
+    fi
+}
+
+check
+
+
```
Comment (wheatli) [inline tools/ci/check-commit.sh]: One empty line in the end is enough

File: tools/ci/check-commit.sh [hunk 3]
```diff
@@ -0,0 +1,122 @@
+# "Copyright [2018] <fisco-bcos>"
+# This script downloads the CMake binary and installs it in $PREFIX directory
+# (the cmake executable will be in $PREFIX/bin). By default $PREFIX is
+# ~/.local but can we changes with --prefix <PREFIX> argument.
+# This is mostly suitable for CIs, not end users.
```
Comment (wheatli) [inline tools/ci/check-commit.sh]: Pls check these comments, is it right??

File: tools/ci/run-clang-format.py (hunk 1 above)
Comment (cyjseagull) [reply tools/ci/run-clang-format.py]: the copyright description has been added

Comment (wheatli) [inline tools/ci/run-clang-format.py]: :+1:

**Comment by wheatli** [comment]
:shipit:

File: tools/ci/check-commit.sh (hunk 2 above)
Comment (cyjseagull) [reply tools/ci/check-commit.sh]: done

File: tools/ci/check-commit.sh (hunk 3 above)
Comment (cyjseagull) [reply tools/ci/check-commit.sh]: already modify this description in https://github.com/FISCO-BCOS/lab-bcos/pull/7/commits/edd15e557faf05912e0deb0a08b1ab81b8a6f448

File: tools/ci/check-commit.sh (hunk 2 above)
Comment (cyjseagull) [inline tools/ci/check-commit.sh]: :+1:<a href='#crh-update-none'></a>

File: tools/ci/check-commit.sh (hunk 3 above)
Comment (cyjseagull) [inline tools/ci/check-commit.sh]: :+1:<a href='#crh-update-complete'></a>

**Pull request merged**